### PR TITLE
Added disinfection system

### PR DESCRIPTION
**Pull request by lucasAxberg**
Added system for calculating the disinfected area + system for higlighting the disinfected area

### CODE REVIEW

File: VR_RV/Assets/Scenes/Gameplay Scenes/Gameplay Scene 4/Scripts/MeshPaint.cs [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+        }
+
+        // Calculate starting point and max width and height
+        int x_starting_value = (int)x_values_in_pixels.Min();
+        int y_starting_value = (int)y_values_in_pixels.Min();
+        int max_width_in_pixels = (int)x_values_in_pixels.Max() - x_starting_value;
+        int max_height_in_pixels = (int)y_values_in_pixels.Max() - y_starting_value;
+        
+        
+        // Printing x and y coodrinates
```
Comment (tramvaj99) [inline VR_RV/Assets/Scenes/Gameplay Scenes/Gameplay Scene 4/Scripts/MeshPaint.cs:104]:
```suggestion
        // Printing x and y coordinates
```

File: VR_RV/Assets/Scenes/Gameplay Scenes/Gameplay Scene 4/Scripts/MeshPaint.cs [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+            
+            // Adjusts the angle to be in the interval (-PI < v < PI)
+            while (delta_theta > Mathf.PI) delta_theta -= Mathf.PI * 2;
+            while (delta_theta < -Mathf.PI) delta_theta += Mathf.PI * 2;
+            
+            // Adds the new angle to the sum of angles
+            sum_of_angles += delta_theta;
+        }
+
+        // If the sum of angles are PI (with a margin of 0.01), return true
```
Comment (tramvaj99) [inline VR_RV/Assets/Scenes/Gameplay Scenes/Gameplay Scene 4/Scripts/MeshPaint.cs:172]:
```suggestion
        // If the sum of angles is PI (with a margin of 0.01), return true
```

File: VR_RV/Assets/Scenes/Gameplay Scenes/Gameplay Scene 4/Scripts/MeshPaint.cs [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+        if (Mathf.Abs(sum_of_angles) + 0.01f > Mathf.PI) return true;
+        return false;
+        
+    }
+
+    bool Array_Contains(ContactPoint[] contact_array, ContactPoint compared_contact)
+    {
+        // Loops through all the collision vertecies
```
Comment (tramvaj99) [inline VR_RV/Assets/Scenes/Gameplay Scenes/Gameplay Scene 4/Scripts/MeshPaint.cs:180]:
```suggestion
        // Loops through all the collision vertices
```

**Review by tramvaj99 — CHANGES_REQUESTED** [review]
Looks great, I just acted as spellcheck for some comments.

**Review by tramvaj99 — APPROVED** [review]
Great!

**Review by abbolisar — APPROVED** [review]
**a**

**Pull request merged**